### PR TITLE
fix(twitter-icon): updated twitter icon and fixed unrelated changes

### LINKED ISSUE #837: 🐞 [BUG] - Unrelated files changes --PWA thing
### Description

Avoid modifying files related to the PWA. See if those changes are necessary. can be done at build process once. these should not come in git history

### Current behaviour

these modify on every build or running dev server ----seems unrelated changes

### Expected behaviour

These should not be changed ideally.. if getting changes then bare minimum should not come in git history

### Reproduction steps

```bash
Just run the server.
```

### Screenshots

Screenshot 2024-07-09 at 10 38 55 PM

### Logs

_No response_

### Browsers

Firefox, Chrome, Safari, Microsoft Edge, Opera

### OS

Mac

### Are you contributing for a program ?

Other

### Code of Conduct

- [X] I have read and agree to the Contributing Guidelines of this project.

### LINKED ISSUE #853: Change Twitter Symbol to X in footer section
I would like to propose a change to update the Twitter symbol to "X" to reflect Twitter's recent rebranding. This change ensures consistency with the platform's new identity and avoids any potential confusion for users. 
please assign this issue to me
Screenshot 2024-10-05 025051

### PR DESCRIPTION
**Pull request by Namit2111**
## Related Issue

Closes: #837 
Closes: #853 

## Description of Changes

- [ ✔] My code adheres to the established style guidelines of this project.
- [ ✔] I have conducted a self-review of my code.
- [✔ ] I have included comments in areas that may be difficult to understand.
- [✔ ] I have made corresponding updates to the project documentation.
- [✔ ] My changes have not introduced any new warnings.

### CODE REVIEW

File: .gitattributes
Comment (Sanchitbajaj02) [inline .gitattributes:1]: What is the need of .gitattributes?

**Review by Sanchitbajaj02 — CHANGES_REQUESTED** [review]

**Comment by Sanchitbajaj02** [comment]
Hi @Namit2111 
How is your PR fixes the issue #837 ? Can you give a brief about it?

File: .gitattributes
Comment (Namit2111) [reply .gitattributes:1]: It is required to apply the behavior on every system, and it removes the need for locally configuring their Git.

**Comment by Namit2111** [comment]
> Hi @Namit2111 How is your PR fixes the issue #837 ? Can you give a brief about it?

The solution sets LF line endings by default for all systems using .gitattributes, preventing automatic conversions caused by OS differences.

The automatic conversion of files was adding files every time in the git history.

**Review by Sanchitbajaj02 — APPROVED** [review]
LGTM

**Pull request merged**